versioned_docs/version-3.0.0-beta/billing/plans.md

```diff
@@ -0,0 +1,72 @@
+---
+id: plans
+title: Plans
+sidebar_label: Plans
+slug: /billing/plans
+---
+
+Spheron platform currently offers below two package for its user:
+
+1. <b>Starter</b>
+2. <b>Pro</b>
+
+Once a user has successfully logged in, he is granted access to Spheron's Starter pack Which is accessible for 30 days, starting right away.
+
+### What features does the Spheron starter pack offer?
+
+Spheron’s starter pack is accessible for 30 days, starting right away. It provides the following features.
+
+- <b>100 GB bandwidth</b>
+- <b>1 Concurrent Build</b>
+- <b>100 Deployments / Day</b>
+- <b>10 Domain Limit</b>
+
+### How do I upgrade to the Pro package?
+
+**Pro** is Spheron's exclusive package. To upgrade to Pro, go to the **Billing** section of your organization and follow the steps mentioned [here](/quick-start/upgrade-plan#upgrade-plan).
+
+:::info
+
+Make sure you have connected your wallet and have set your allowance.
+
+:::
+
+### Upgrade Plan
+
+Users can upgrade or downsize their team in Spheron's Pro membership model to fit their company needs.
+
+#### PRO Package
+
+Currently, the <b>Pro</b> subscription plan costs $18 per month. Covering in features like
+
+- <b>500 GB Bandwidth</b>
```

Comment:
these are wrong details, please check the doc for correct plan details:
https://docs.google.com/document/d/1pSgJHS_Xc1swpH1RTtHPrHISgLB81bC94zYrhMSFhXs/edit#heading=h.twaq50ac05wd